lab_1/main.py

```diff
@@ -11,7 +12,18 @@ def tokenize(text: str) -> list or None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
-    pass
+    if isinstance(text,str):
+        smbl_in_txt = ''
+        text = text.lower()
+        for symbol in text:
+            if symbol.isalpha() == True or symbol == ' ' or symbol == '\n':
+                smbl_in_txt += symbol
+        tokens = re.findall(r'\w+', smbl_in_txt)
+        for token in tokens:
+            if token == 'º':
```

Comment:
can you avoid checking this (º) character in separate statement?
try adding this character into your regular expression